Topics/Tech_Stacks/JSONRPC.md

```diff
@@ -0,0 +1,25 @@
+### JSON-RPC
+
+## Introduction
+JSON-RPC is a an approach to building APIs for communication between different software systems. Let's compare it to REST, which is the most common way of developing APIs. JSON-RPC is not as popular, but it offers its own advantages. Like REST, JSON-RPC uses JSON for its data format and uses HTTP as a transport protocol. JSON-RPC focuses on remote procedure calls whereas the more popular REST is more focused on resource manipulation. 
+
+This makes it so JSON-RPC is a better option in scenarios where remote functions have to be called on a server that require an action result. It's most useful for when complex calculations or calling remote procedures on the server are needed. For example, a good time to JSON-RPC is transfering money between bank account on a remote system. 
+
+REST is generally more useful in situations where actions such as adding data to databases or updating information are needed due to its ability to easily perform CRUD operations.
+
```

Comment:
This is a good introduction to JSON RPC. Since this is a brief overview, you should consider linking to external resources in case the reader wants to learn more in depth about JSON-RPC and why they might choose it over REST.